tgui/packages/tgui/interfaces/ProduceConsole.js

```diff
@@ -152,39 +152,28 @@ const CheckoutTab = (props, context) => {
                   fluid
                   icon="plane-departure"
                   content="Purchase"
-                  tooltip={multiline`
+                  tooltip={`
                   Your groceries will arrive at cargo,
                   and hopefully get delivered by them.
+                  ${cargo_cheaper ? 'Cheaper than express delivery.' : ''}
                   `}
                   tooltipPosition="top"
                   onClick={() => act('purchase')}
                 />
               </Stack.Item>
             )}
-            {!!ltsrbt_available && (
-              <Stack.Item grow textAlign="center">
-                <Button
-                  fluid
-                  icon="shuttle-van"
-                  content="Deliver"
-                  tooltip={multiline`
-                  Your groceries will arrive to one of
-                  the on-station built LTSRBT devices.
-                  `}
-                  tooltipPosition="top"
-                  onClick={() => act('ltsrbt_deliver')}
-                />
-              </Stack.Item>
-            )}
             <Stack.Item grow textAlign="center">
               <Button
                 fluid
                 icon="parachute-box"
                 color="yellow"
                 content="Express"
-                tooltip={multiline`
-                Sends the ingredients instantly,
-                but locks the console longer and increases the price!
+                tooltip={`
+                Sends the groceries instantly${
```

Comment:
I didn't catch this when I worked on it, but please remove the mentions of groceries/ingredients